bonfire/utils.py

```diff
@@ -390,6 +390,14 @@ def get_dependencies(items, optional=False):
         log.debug("clowdapp '%s' has %s: %s", name, key, list(dependencies))
         deps_for_app[name] = dependencies
 
+    for item in items:
+        name = item.get("metadata", {}).get("name")
+        bonfire_deps = item.get("metadata", {}).get("annotations", {}) \
+            .get("bonfire.dependencies", "").split(',')
+        filtered_bonfire_deps = list(filter(lambda x: x != '', bonfire_deps))
```

Comment:
What you have is fine and works but you could be a bit more idiomatic by using a list comprehension

```
filtered_bonfire_deps = [dep for dep in bonfire_deps if dep]
```

Reply:
i did notice there was more list comprehensions than lambda's present. I'll make those changes so it fits in better